lib/quantization/src/encoded_vectors_binary.rs

```diff
@@ -575,7 +554,11 @@ impl<TBitsStoreType: BitsStoreType, TStorage: EncodedStorage>
         }
     }
 
-    fn encode_two_bits_value(value: f32, mean: f32, sd: f32) -> (bool, bool) {
+    fn encode_two_bits_value(
+        vector: &[f32],
+        i: usize,
```

Comment:
why not just `f32`? It is only 4 bytes on stack, instead of 4 + 4 for current version